### ISSUE TITLE
Test building vs fetching of content-build

### ISSUE THREAD

**Issue by JoeTice** [department-of-veterans-affairs/va.gov-team#60689]

**Comment by CBonade** [comment]
6/9 - attempting to implement this resulted in an error of an undefined value while trying to fetch the Drupal cache. Search for a fix or a workaround was unfruitful. Heading out to vacation handing off the information on progress for this to Holden/Peter while i'm out.

6/22 - attempts while I was out went a different direction but this resurfaced as a piece of information we needed to have. Peter ran a test today and when it deployed, it showed the proper values, however the API URL had a typo so we could not verify a proper communication with the API. When he fixed it and redeployed, the pod/cluster/container/<insert other devops term here> ran out of space and was not able to finish deploying. Cyril/Rachal/Peter were working at a solution for this but it remains unsolved currently. Blocked awaiting devops un-blockage.

**Comment by CBonade** [comment]
In our testing since then, building content-build at runtime seems to be an unnoticeable difference in time compared to fetching an archived version. As we were able to solve all errors related to this build process and any existing errors are unrelated to this build, this ticket is considered a success and ready to close.

**Comment by pjhill** [comment]
Agreed, switching to the real time build doesn't appear to have a major impact and some of the unexpected behaviors have disappeared. It seems like this is the approach to take.
